channel.go

```diff
@@ -131,7 +131,7 @@ func (c *serialChannel) wait() error {
 	var events [1]unix.EpollEvent
 
 	fd := c.serialConn.Fd()
-	if fd <= 0 {
+	if fd == 0 {
```

Comment:
is this change needed? :worried:

Reply:
Looks safe since https://golang.org/pkg/os/#File.Fd returns a `uintptr`. Don't need to worry about negative values.

Reply:
yup. that was the complaint by the linter.

Reply:
thanks @sboeuf @ganeshmaharaj